src/components/Questionnaire/QuestionnaireForm.tsx

```diff
@@ -402,6 +402,7 @@ export function QuestionnaireForm({
                     );
                   }}
                   disabled={isPending}
+                  className={cn(buttonVariants({ variant: "destructive" }))}
```

Comment:
what's the point of using buttonVariants along with cn when button directly allows passing variant?